core/src/main/java/com/linecorp/armeria/common/logging/RequestContextExporterBuilder.java

```diff
@@ -134,16 +122,17 @@ public RequestContextExporterBuilder responseHeader(CharSequence headerName) {
     public RequestContextExporterBuilder responseHeader(CharSequence headerName, String alias) {
         requireNonNull(headerName, "headerName");
         requireNonNull(alias, "alias");
-        return addResponseHeader(toHeaderName(headerName), alias);
-    }
-
-    private RequestContextExporterBuilder addResponseHeader(AsciiString headerKey, String alias) {
-        resHeaders.add(new ExportEntry<>(headerKey, alias));
+        defaultExportGroupBuilder.responseHeader(headerName, alias);
         return this;
     }
 
-    private static AsciiString toHeaderName(CharSequence name) {
-        return HttpHeaderNames.of(requireNonNull(name, "name").toString());
+    /**
+     * Specifies a prefix of the default export group.
+     */
+    public void setDefaultExportPrefix(String defaultExportPrefix) {
```

Comment:
- How about just `prefix`? IIUC, the following example:
  ```java
                RequestContextExporter
                        .builder()
                        .setDefaultExportPrefix("foo")
                        .addExportGroup(
                                ExportGroup
                                        .builder()
                                        .keyPattern("client.*")
                                        .build()
                        )
  ```
  .. does not make `client.*` have `foo` prefix, so it'll be confusing to say `foo` is the default prefix.
- `set` could be omitted.

Reply:
Yeah, `.setDefaultExportPrefix(String)` should be renamed to `.prefix(String)`.